src/plugins/explore/public/application/hooks/editor_hooks/use_change_query_editor/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './use_change_query_editor';
```

Comment:
guess i'm little lost we have an index file but it only references the same file by name can we shorten this filepath but just reference the hooks with file names vs having a folder for each?

Reply:
its more organizational, there are like 8 hooks in this folder, each with an unit test